core/src/main/scala/akka/persistence/postgres/config/AkkaPersistenceConfig.scala

```diff
@@ -57,6 +57,22 @@ class SnapshotTableConfiguration(config: Config) {
   override def toString: String = s"SnapshotTableConfiguration($tableName,$schemaName,$columnNames)"
 }
 
+class TagsTableColumnNames(config: Config) {
+  private val cfg = config.asConfig("tables.tags.columnNames")
+  val name: String = cfg.asString("name", "name")
+
+  override def toString: String = s"TagsTableColumnNames($name)"
+}
+
+class TagsTableConfiguration(config: Config) {
+  private val cfg = config.asConfig("tables.tags")
+  val tableName: String = cfg.asString("tableName", "tags")
+  val schemaName: Option[String] = cfg.as[String]("schemaName").trim
```

Comment:
I thinnk that you can use `cfg.asOptionalNonEmptyString`

Reply:
or : `cfg.as("schemaName").trim`

Reply:
👍